<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentSchemeEditor</class>
 <widget class="QDialog" name="PaymentSchemeEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Протокол</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата окончания договора</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLineEdit" name="edtPostAddress"/>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="edtContractName"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblContractName">
     <property name="text">
      <string>Наименование протокола</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLineEdit" name="edtSubjectContract"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lblSubjectContract">
     <property name="text">
      <string>Предмет договора</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lblStatus">
     <property name="text">
      <string>Статус учреждения</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="CDateEdit" name="edtDate"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="lblContractNumber">
     <property name="text">
      <string>Номер протокола</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="CPersonComboBoxEx" name="cmbPerson"/>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="lblContractSum">
     <property name="text">
      <string>Сумма договора</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="COrgComboBox" name="cmbOrgId">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnSelectOrgName">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="2">
    <widget class="CDateEdit" name="edtEndDate"/>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="cmbStatus">
     <property name="minimumSize">
      <size>
       <width>234</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Страховая компания</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Медицинский центр</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Благотворительное общество</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Другая организация</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="lblDate">
     <property name="text">
      <string>Дата заключения договора</string>
     </property>
     <property name="buddy">
      <cstring>edtDate</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="edtContractNumber">
     <property name="minimumSize">
      <size>
       <width>234</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLineEdit" name="edtContractSum">
     <property name="minimumSize">
      <size>
       <width>234</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="lblOrgName">
     <property name="text">
      <string>Наименование организации</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>Главный исследователь</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lblPostAddress">
     <property name="text">
      <string>Почтовый адрес</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QCheckBox" name="chkEnrollment">
     <property name="text">
      <string>Набор закрыт</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="edtNumber"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblNumber">
     <property name="text">
      <string>Номер договора</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>COrgComboBox</class>
   <extends>QComboBox</extends>
   <header>OrgComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>OrgStructComboBoxes.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PaymentSchemeEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PaymentSchemeEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
